### PR TITLE
Update jira-modeling-rule

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Remove unknown char from the file

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/ModelingRules/JiraEventCollector/JiraEventCollector.yml [hunk 1]
```diff
@@ -1,4 +1,53 @@
...
+      "created": {
+        "type": "string",
+        "is_array": false
+      },
+      "category": {
+        "type": "string",
+        "is_array": false
+      },
+      "objectItem": {
+        "id": {
```
Comment (anara123) [inline Packs/Jira/ModelingRules/JiraEventCollector/JiraEventCollector.yml:33]: @ShahafBenYakir do we support nested layers?

File: Packs/Jira/Integrations/JiraEventCollector/JiraEventCollector.yml [hunk 2]
```diff
@@ -43,6 +32,18 @@ configuration:
   name: proxy
   required: false
   type: 8
+- defaultvalue: jira
+  additionalinfo: "For the dataset name for example: jira"
+  display: Vendor name
+  name: vendor
+  required: false
+  type: 0
```
Comment (ShahafBenYakir) [inline Packs/Jira/Integrations/JiraEventCollector/JiraEventCollector.yml:40]: Need to define a standard here. In other cases, we only gave the product as a param.
Correct me if I am wrong here @yaakovi

**Review by MosheEichler — APPROVED** [review]

File: Packs/Jira/ReleaseNotes/2_0_2.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Modeling Rules
+- **JiraEventCollector**
+- Improved implementation of the modeling rules.
```
Comment (amshamah419) [inline Packs/Jira/ReleaseNotes/2_0_2.md:4]:
```suggestion
- Improved the implementation for the modeling rules.
```

File: Packs/Jira/ReleaseNotes/2_0_2.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Modeling Rules
+- **JiraEventCollector**
+- Improved the implementation for the modeling rules.
```
[the remark this answers is not in the document]
Comment (MosheEichler) [reply Packs/Jira/ReleaseNotes/2_0_2.md:4]: Yes it is

**Pull request merged**